apps/asap-server/test/routes/events.test.ts

```diff
@@ -1,12 +1,14 @@
 import supertest from 'supertest';
 import { appFactory } from '../../src/app';
 
+jest.mock('../../src/utils/validate-token');
```

Comment:
@johnytiago I think we should get rid of this and try to use some sort of DI here: https://github.com/yldio/asap-hub/pull/576/commits/3680d9973a2969ee0aeeb98ca453d09f275f8446#diff-22b746f2e428e4960dd3cebd09513b984e202dc47ee52548a213cb56843dbb4cR7, what do you think?

Reply:
Agreed 👍🏽